bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md

```diff
@@ -0,0 +1,107 @@
+# Paper review-Perception Aware Model Predictive Control for Quadrotors 
+
+Authors - Davide Falanga, Philipp Foehn, Peng Lu, and Davide Scaramuzza
+
+## Why is this important?
+
+As we are advancing into a world of autonomous control of vehicles, there have been some key developments that have accelerated the research in the domain. One of these, is the Perception Aware Model Predictive Control (PAMPC) for Quadrotors, which I am going to review in this document. The focus of this review is to summarize the Model Predictive Control (MPC) pipeline used along with its cost function and control constraints.
+
+## Introduction
```

Comment:
This part clearly states the relevance and emerging importance of integrating perception with control. Although it would be better if the writer can explain what does perception means in this context to give reader the right understanding. It will also be helpful if the writer can could include a brief overview or examples of previous methods and their limitations to contrast why PAMPC is beneficial.